test/parallel/test-fs-write-stream-autoclose-option.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var path = require('path');
+var fs = require('fs');
+
+var file = path.join(common.tmpDir, 'write-autoclose-opt1.txt');
+common.refreshTmpDir();
+var stream = fs.createWriteStream(file, {flags: 'w+', autoClose: false});
+stream.write('Test1');
+stream.end();
+stream.on('finish', function() {
+  process.nextTick(function() {
+    assert(!stream.closed);
+    assert(stream.fd);
+    next();
+  });
+});
+
+function next() {
```

Comment:
You can do a similar thing here: `const next = common.mustCall(function() { ... });`

It's almost never wrong to use `common.mustCall`, use it abundantly.